include/mls/credential.h

```diff
@@ -47,6 +47,18 @@ struct X509Credential
   SignatureScheme _signature_scheme;
 };
 
+struct UserInfoVCCredential
+{
+  UserInfoVCCredential() = default;
+  explicit UserInfoVCCredential(bytes userinfo_vc_jwt);
```

Comment:
We discussed this in our sync meeting, and it was decide that this would not be best because it would change the way that a user called this function.  They would need to know to pass the  rvalue reference.